CHANGELOG.md

```diff
@@ -68,6 +68,9 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 - Added a `language` field to the `Account`. Consequently, the `account` and
   `accountList` API responses now include this field. The `insertAccount` and
   `updateAccount` GraphQL API endpoints are also updated to support the field.
+- Added `common_sign-in_logic` to handle `sign-in` GraphQL API logic.
```

Comment:
This is not necessary because the method is not for users.

Reply:
I'll delete it.